Docs/source/usage/parameters.rst

```diff
@@ -166,6 +166,11 @@ Setting up the field mesh
 
     Note: in development; currently allowed value: ``2 2 2``.
 
+* ``warpx.end_fine_patch_step`` (`integer`, default: ``-1``)
+    When using mesh refinement, this can be used to remove the finest level at a given timestep.
```

Comment:
I would also add a couple of lines to explain what happens to the timestep duration.